package.json

```diff
@@ -115,7 +115,8 @@
     "tiny-lr": "^1.0.3",
     "tree-sync": "^1.1.4",
     "walk-sync": "^0.3.0",
-    "yam": "0.0.22"
+    "yam": "0.0.22",
+    "yarn": "^0.17.8"
```

Comment:
I can move it to `devDependencies` to avoid shipping it unnecessarily. We want to use it because it saves us a ton of time in our test suite, I'll set it up to fall back to `npm` for older versions of Node. (Which already works.) That meets our Node LTS commitment.

But, I just realized that we can use this same approach to make `ember new/addon/init` faster for our users. So I may leave it in `dependencies` and wire it into those as well.